src/modules/app/components/GeneralHeader.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import locationIcon from '../assets/location-icon.svg'
+import Header from 'modules/layout/components/Header'
+
+class GeneralHeader extends React.Component {
```

Comment:
I think we don't really need separate components here. They just hold a small configuration.
We can pass this directly when using it: `<Header actionItems={[{href: '/', iconSrc: locationIcon}]} />`
Less components -> less tests :)

Reply:
The test for that component will really just be a snapshot. 
The thing is: I'd like to have that every route determines its layout itself. Therefore I'd like to strip the RouterFragment.js in app module and remove the layout wrappers because they don't really belong there imo. But that way we would have these wrapper components in the routePages again. Opinions on that? :D

Reply:
I don't see the connection between the GeneralHeader and the Layout/Wrapper components thing :/

We could add a HOC which builds a layout around a components. I think that would avoid the WrapperComponents